opwen_email_server/integration/celery.py

```diff
@@ -73,6 +74,17 @@ def inbound_store(resource_id: str) -> None:
     action(resource_id)
 
 
+@celery.task(ignore_result=True)
+def process_service(resource_id: str) -> None:
```

Comment:
I believe naming this after the concrete service it's implementing ("process_secho_service_email" or something like that) may be clearer... I think we'll need one of these for each email service we're implementing.

Reply:
I see! will do!